core/base/libs/socrate/socrate/system.py

```diff
@@ -97,6 +97,8 @@ def set_env(required_secrets=[], log_filters=[], log_file=None):
     for secret in required_secrets:
         os.environ[f'{secret}_KEY'] = hmac.new(bytearray(secret_key, 'utf-8'), bytearray(secret, 'utf-8'), 'sha256').hexdigest()
 
+    os.system('find / -xdev -type f -name \*.pid -print -delete')
```

Comment:
Shouldn't we know where the pid files are located other than /?

Reply:
The idea was to make it future proof. By adding -xdev we ensure we only crawl "the image" so it's reasonably fast... and only done once at startup anyway.

Reply:
Oh and another thing is that you can't list "non existing" directories as find arguments, it will error out

Reply:
Yeah. Sure. But I really like to stick to /run here without xdev.
This is where pid files have to reside according to LFS.
Deleting files outside doesn't feel right.

Reply:
As seen on the PR, postfix is at /queue/pid/master.pid ... so do you want me to restrict this to /run and revert the change on postfix?

Reply:
Ah that's bad. But /queue is not scanned when using `-xdev` so it won't be found either.
```
b9cf64850518:/app# ls /queue/pid/
inet.10025     master.pid     unix.cleanup   unix.flush     unix.outclean  unix.smtp
inet.smtp      unix.bounce    unix.defer     unix.lmtp      unix.retry     unix.trace
b9cf64850518:/app# find / -xdev -name '*.pid' 2>/dev/null
b9cf64850518:/app# find / -name '*.pid' 2>/dev/null
/queue/pid/master.pid
```

Reply:
Good point, that's bbed508e